libraries/consul.rb

```diff
@@ -18,5 +18,23 @@ def self.setup_consul(options)
         options[:token] = options[:consul_token]
       end
     end
+
+    def self.update_backup_url(options)
+      require 'diplomat'
+      require 'diplomat/version'
+      return unless options[:consul_backup_url]
+
+      Diplomat.configure do |config|
+        config.url = @options[:consul_backup_url]
+      end
+    end
+
+    def self.reset_url
+      require 'diplomat'
+      require 'diplomat/version'
+      Diplomat.configure do |config|
+        config.url = 'http://localhost:8500'
```

Comment:
What about using Diplomat::Configuration.parse_consul_addr here, instead of hardcoding localhost?
https://github.com/WeAreFarmGeek/diplomat/blob/master/lib/diplomat/configuration.rb#L12

Reply:
Good idea